### PR TITLE
[ground] Distance measuring during flight, Ubuntu18.04 update + time

### PR DESCRIPTION
**Pull request by dewagter**
- controlpanel entry (https://github.com/tudelft/paparazzi/pull/58)
 - time counter with buttons (https://github.com/tudelft/paparazzi/pull/59)
 - python Images/PIL fix

### CODE REVIEW

**Comment by dewagter** [comment]
@danielwillemsen

File: sw/ground_segment/python/distance_counter/distance_counter.py [hunk 1]
```diff
@@ -86,11 +93,27 @@ def OnPaint(self, e):
         dc.DrawText("INS Packets:" + str(self.msg_count),2,2)
         dc.DrawText("Data: " + str(self.ins_msg_x) + ", " + str(self.ins_msg_y) + ", " + str(self.ins_msg_z) + ".",2,22)
         dc.DrawText("Distance: " + str(round(float(self.distance)/1.0,2)) + " m",2,22+20)
+        dc.DrawText("Time elapsed: " + str(self.time_elapsed) + "s",2,22+20+20)
+        print("jo")
```
Comment (matteobarbera) [inline sw/ground_segment/python/distance_counter/distance_counter.py:97]:
```suggestion
        # print("jo")
```

File: sw/ground_segment/python/distance_counter/distance_counter.py [hunk 2]
```diff
@@ -86,11 +93,27 @@ def OnPaint(self, e):
...
+        if self.running:
+            dc.DrawText("Counter running", 150, 22+20)
+        else:
+            dc.DrawText("Counter paused", 150, 22+20)
+
+    def onStartStop(self, event):
+        print(self.running)
```
Comment (matteobarbera) [inline sw/ground_segment/python/distance_counter/distance_counter.py:104]:
```suggestion
        # print(self.running)
```

File: sw/ground_segment/python/distance_counter/distance_counter.py [hunk 3]
```diff
@@ -17,19 +17,20 @@
 # along with paparazzi.  If not, see <http://www.gnu.org/licenses/>.
 #
 
+# This is not the main script. Run dist.py to have a distance counter.
+
 import wx
 import sys
-import os
+import os
```
Comment (dewagter) [inline sw/ground_segment/python/distance_counter/distance_counter.py:24]:
```suggestion
import os
```

**Comment by danielwillemsen** [comment]
Thanks for removing the prints, Matteo. I still think there is one more thing that needs to be fixed: this version won't work if the rotorcraft status message has a different number of fields, which it might have. I'll fix this tomorrow at the latest.

**Comment by danielwillemsen** [comment]
@dewagter As I dont have push authorization to python_distance_p3_time branch (which is on the paparazzi repo?), I created a pull request to merge my fix into that branch, see  #2522 . Everything seems alright with me now.

**Comment by dewagter** [comment]
Thank you @danielwillemsen .

**Pull request merged**